doc/how_to_build_linux.md

```diff
@@ -97,11 +97,12 @@ OpenToonzPROFILES="$HOME/.config/OpenToonz/stuff/profiles"
 TOONZCACHEROOT="$HOME/.config/OpenToonz/stuff/cache"
 TOONZCONFIG="$HOME/.config/OpenToonz/stuff/config"
 TOONZFXPRESETS="$HOME/.config/OpenToonz/stuff/projects/fxs"
-TOONZLIBRARY="$HOME/.config/OpenToonz/stuff/projects/library"
+TOONZLIBRARY="$HOME/.config/OpenToonz/stuff/library"
 TOONZPROFILES="$HOME/.config/OpenToonz/stuff/profiles"
 TOONZPROJECTS="$HOME/.config/OpenToonz/stuff/projects"
 TOONZROOT="$HOME/.config/OpenToonz/stuff"
 TOONZSTUDIOPALETTE="$HOME/.config/OpenToonz/stuff/studiopalette"
+TOONZFXPRESETS="/home/USERNAME/.config/OpenToonz/stuff/fxs"
 EOF
 ```
```

Comment:
Hi @blackwarthog 
I completely uninstalled opentoonz and cloned it again and went through the installation. but it seems that we still need the `sysvar.ini`
when I tried to run opentoonz I got:
```
rdi@rdi-adam:build (patch-1)$ LD_LIBRARY_PATH=./lib/opentoonz:$LD_LIBRARY_PATH ./bin/OpenToonz_1.1
varName:OpenToonzPROFILES TOONZROOT not set...
varName:OPENTOONZROOT TOONZROOT not set...
varName:OpenToonzPROFILES TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZPROFILES TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZPROFILES TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZPROFILES TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZPROFILES TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZCONFIG TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZCONFIG TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
varName:TOONZROOT TOONZROOT not set...
QSettings::setValue: Empty key passed
Segmentation fault
```